src/commands/pwd_command.c

```diff
@@ -0,0 +1,16 @@
+#include <commands/commands.h>
+#include <unistd.h>
+#include <libft.h>
+#include <stdio.h>
+
+int	pwd_command(t_output_stdout output)
+{
+	char	buffer[4096];
+	int		len;
+
+	getcwd(&buffer[0], sizeof(buffer));
```

Comment:
Might be a good idea to check that the `getcwd()` call succeeds? Something like:
`if (getcwd(&buffer[0], sizeof(buffer) == ERR)
         handle_errors(error_code);`